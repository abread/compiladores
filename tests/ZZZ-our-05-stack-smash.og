auto f() {
	return ("oh", "my"), ("w", 0, "t"), "a", ("tup", 1, 3), 1.2;
}
public int og() {
	ptr<real> ptrreal = nullptr;
	auto a = 43, ("asd", 4, ptrreal), 56.0;

	a@2@3 = [10];

	for int i = 0; "moar conds", (1.5, (2, (3, 4))@2), i < 10; i = i + 1 do {
		a@2@3[i] = 0.1 * i;
	}

	{
		auto b = f();
		("oh", "my", (("we", "like", (7, ("up", 1.3), "s"))@3@2@2, "wayy t", 0.0, "much")@1)@3 = 5;
	}


	for int i = 0; i < 10; i = i + 1, "moar incrs", (1.5, (2, (3, 4))@2) do {
		writeln a@2@3[i];
	}

	return 0;
}

