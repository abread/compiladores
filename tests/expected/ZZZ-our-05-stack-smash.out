0
1E-1
2E-1
3E-1
4E-1
5E-1
6E-1
7E-1
8E-1
9E-1
